tests/e2e/test_e2e.py

```diff
@@ -368,3 +368,10 @@ def test_all_rules_have_added_in_version_info(self, robocop_instance):
         robocop_instance.reload_config()
         without_version = {rule.name for rule in robocop_instance.rules.values() if not rule.added_in_version}
         assert without_version == set()
+
+    def test_all_community_rules_should_be_disabled_by_default(self, robocop_instance):
```

Comment:
Maybe the same for built-in rules but with enabled?

Reply:
I will try to implement such test. After I created the same type of the tests I have noticed that it also matches rules that are disabled due to version (ie ``nested-for-loop`` in RF 6). And I found the bug thanks for that - ``if-can-be-used`` for some reason is only enabled for 4.* and not for 5, 6 etc versions.

Reply:
Tests always reveal the true identity :D